src/components/Dashboard/SettingsMenu.jsx

```diff
@@ -5,6 +5,7 @@ import MenuItem from '@material-ui/core/MenuItem';
 import IconButton from '@material-ui/core/IconButton';
 import SettingsOutlineIcon from 'mdi-react/SettingsOutlineIcon';
 import Link from '../../utils/Link';
+import isLoggedIn from '../../utils/isLoggedIn';
```

Comment:
We can use the `withUser` function to have access to the `user` object instead of `isLoggedIn`.

```jsx
import { withUser } from '../../utils/AuthContext';

function SettingsMenu({ user }) {
  // disabled={!user}
}

export default withUser(SettingsMenu);
```